### ISSUE TITLE
Labels "opaque" blending does not persist after click to canvas

### ISSUE THREAD

**Issue by GenevieveBuckley** [napari/napari#5319]
## 🐛 Bug

The labels "opaque" blending mode does not stay opaque after the user clicks on the 3D canvas.

## To Reproduce

1. Set up some example data with labels
```python
import napari
from skimage.filters import gaussian, threshold_otsu
from skimage.measure import label

# Open demo data
viewer = napari.Viewer()
viewer.open_sample('napari', 'cells3d')

# Create labels
nuclei = viewer.layers[-1].data
smoothed = gaussian(nuclei, sigma=2)
thresh = threshold_otsu(smoothed)
mask = smoothed > thresh
labels = label(mask)
viewer.add_labels(labels)

# Switch to 3d view mode and turn on opaque blending mode
viewer.dims.ndisplay = 3
viewer.layers[-1].blending = 'opaque'
```
![napari-labels-opaque-blending](https://user-images.githubusercontent.com/30920819/200716378-91fd671e-e77d-4c31-86b1-8edd3ef48d35.png)

2. Click on the napari canvas and observe the change to the label blending. The labels display becomes slightly more transparent.

![napari-labels-opaque-blending-after-click](https://user-images.githubusercontent.com/30920819/200716518-27c239ee-1fa1-4b99-a3df-269141b57978.png)

Here's a gif, switching between the view before & after clicking the canvas:

![napari-opaque-blending-mode-on-click](https://user-images.githubusercontent.com/30920819/200718032-ead68116-2c53-4d61-b249-6d9dc1ac93bc.gif)

## Expected behavior

I expected that clicking on the canvas should not affect the transparency/display of a labels layer.

## Environment

 - Please copy and paste the information at napari info option in help menubar here:
 
napari: 0.4.16
Platform: macOS-12.3.1-arm64-arm-64bit
System: MacOS 12.3.1
Python: 3.9.13 | packaged by conda-forge | (main, May 27 2022, 17:00:33) [Clang 13.0.1 ]
Qt: 5.15.2
PyQt5: 5.15.4
NumPy: 1.22.4
SciPy: 1.9.1
Dask: 2022.9.1
VisPy: 0.10.0

OpenGL:
- GL version: 2.1 Metal - 76.3
- MAX_TEXTURE_SIZE: 16384

Screens:
- screen 1: resolution 1440x900, scale 2.0
- screen 2: resolution 1920x1080, scale 1.0

Plugins:
- console: 0.0.6
- empanada-napari: 0.2.3
- napari-clusters-plotter: 0.5.2
- napari-nD-annotator: 0.0.8
- napari-svg: 0.1.6
- napari-tools-menu: 0.1.17
- napari_skimage_regionprops1: 0.5.5
- napari_skimage_regionprops2: 0.5.5
- scikit-image: 0.4.16
- zarpaint: 0.1.1.dev27+gfc48307

 - Any other relevant information:

If you query the blending mode using `viewer.layers[-1].blending` it still reports using the "opaque" blending mode, even after the display change caused by clicking on the canvas.

The change in the labels blending mode looks similar to, but is not quite the same as the "additive" blending mode.

**Comment by Czaki** [comment]
Could you chcek if it still happens in napari 0.4.17rc?

**Comment by psobolewskiPhD** [comment]
I can reproduce this in 0.4.16, but on my end it's fixed in 0.4.17.
@GenevieveBuckley Can you double check with the new live release, 0.4.17?

**Comment by GenevieveBuckley** [comment]
It doesn't seem possible to upgrade napari in the environment I'm actually using (too many conflicts), but I've just created a new empty test environment and can confirm that the "opaque" setting is actually opaque in 0.4.17, unlike the screenshots above from 0.4.16

![napari-opaque-3d-labels](https://user-images.githubusercontent.com/30920819/203216191-d79f58b5-fd4d-40fe-8a88-f9c5be97bc74.png)
